analysis.py

```diff
@@ -37,14 +40,17 @@ def build_analysis():
   xml_data = {}
   
   for fn in os.listdir('XML'):
-    if 'v30' in fn:
+    if 'swp' not in fn:
       with open('XML/' + fn) as f:
         contents = xmltodict.parse(f.read())
 
         values = []
         for col_name, path in useful_fields:
-          values.append(get_value(contents, path))
-
+          try:
+            values.append(get_value(contents, path))
+            print(values)
```

Comment:
Probably don't want to print the values each time - this sort of output adds lots of processing overhead in python. Also this print here is a python3 print. I'm using a python2.7 virtualenv.

Reply:
I'm running things in python2.7 but I guess I just got into the habit of writing it like that.
As to why it is there, I like to convince myself that the code is doing what I expect -- but it doens't need to be there at the moment so will remove.